.doc_gen/metadata/dynamodb_metadata.yaml

```diff
@@ -26,8 +26,19 @@ dynamodb_CreateTable:
             - description: Create a table for storing movie data.
               snippet_tags:
                 - dynamodb.python.codeexample.MoviesCreateTable
+    Rust:
+      versions:
+        - sdk_version: 1
+          github: .rust_alpha/dynamodb
+          excerpts:
+            - description:
+              snippet_tags:
+                - dynamodb.rust.create-table
+                - dynamodb.rust.crud-make_table
+                - dynamodb.rust.dynamodb-helloworld-create_table
+                - dynamodb.rust.movies-create_table

```

Comment:
The way you have these stacked looks a little confusing in the output because they're all rendered as if they're in one code file. You can intersperse brief descriptions, which might help. Something like:
```
- description: Create a table with a key.
  snippet_tags:
    - dynamodb.rust.create-table
- description: Create a table of movies.
  snippet_tags:
    - dynamodb.rust-movies-create_table
```

Reply:
I've removed the duplicates.